### ISSUE TITLE
Prefix configuration

### ISSUE THREAD

**Issue by Ultra03** [Miku-Chan-Devs/Miku-Chan#56]
I was thinking for v2 we should allow people to customize the prefix via a command. This could be done with [Enmap](https://www.npmjs.com/package/enmap), or with something else.

**Comment by Incrested** [comment]
Yeah, I'm cool with this mate: Enmap should work fine; check if Heroku will be able to store a decent amount of guild-exclusive data on a free dyno and see, if we do have to upgrade, if we should start self-hosting

**Comment by Incrested** [comment]
Also: we do have everything there. We just have to do a command to edit an exclusive to guild config file
